qtranslate_hooks.php

```diff
@@ -225,6 +225,9 @@ function qtranxf_wpseo_replacements($replacements){
 add_filter('link_description', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 add_filter('the_author', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 
+add_filter('get_terms', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
+add_filter('term_description', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
```

Comment:
Hi, josk79. could you point me to a place with a complimentary calls to `apply_filters('term_description')` in the WP code?